android/quest/src/main/java/org/smartregister/fhircore/quest/ui/family/profile/components/ChangeFamilyHeadBottomSheet.kt

```diff
@@ -16,7 +16,215 @@
+  coroutineScope: CoroutineScope,
+  bottomSheetScaffoldState: BottomSheetScaffoldState,
+  familyMembers: FamilyProfileViewModel.ChangeFamilyMembersHolder?,
+  onSaveClick: (FamilyMemberViewState) -> Unit,
+  modifier: Modifier = Modifier
+) {
+
+  var source by remember { mutableStateOf(familyMembers) }
+  var isEnabled by remember { mutableStateOf(false) }
+  Surface(shape = RoundedCornerShape(topStart = 8.dp, topEnd = 8.dp)) {
+    Column(modifier = modifier.fillMaxWidth()) {
+      Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically,
+        modifier =
+          modifier
+            .fillMaxWidth()
+            .height(IntrinsicSize.Min)
+            .padding(horizontal = 16.dp, vertical = 16.dp)
+      ) {
+        Text(
+          text = stringResource(id = R.string.label_assign_new_family_head),
+          textAlign = TextAlign.Start,
+          fontWeight = FontWeight.Light,
+          fontSize = 20.sp,
+        )
+        Icon(
+          imageVector = Icons.Filled.Clear,
+          contentDescription = null,
+          tint = DefaultColor.copy(0.8f),
+          modifier =
+            modifier.clickable {
+              coroutineScope.launch {
+                if (!bottomSheetScaffoldState.bottomSheetState.isCollapsed)
+                  bottomSheetScaffoldState.bottomSheetState.collapse()
+              }
+            }
+        )
+      }
+      Divider()
+      Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier =
+          modifier
+            .fillMaxWidth()
+            .height(IntrinsicSize.Min)
+            .padding(horizontal = 12.dp, vertical = 18.dp)
+            .background(
+              color = colorResource(id = R.color.background_warning),
+              shape =
+                RoundedCornerShape(
+                  topStart = 8.dp,
+                  topEnd = 8.dp,
+                  bottomStart = 8.dp,
+                  bottomEnd = 8.dp
+                )
+            )
+      ) {
+        Image(
+          painter = painterResource(id = R.drawable.ic_alert_triangle),
+          contentDescription = null,
+          modifier = modifier.padding(horizontal = 12.dp)
+        )
+        Text(
+          text = stringResource(id = R.string.alert_message_abort_operation),
+          textAlign = TextAlign.Start,
+          fontWeight = FontWeight.Medium,
+          fontSize = 16.sp,
+          modifier = modifier.padding(vertical = 12.dp)
+        )
+      }
+      Text(
+        text = stringResource(id = R.string.label_select_new_head),
+        modifier = modifier.padding(horizontal = 12.dp),
+        textAlign = TextAlign.Start,
+        fontWeight = FontWeight.Light,
+        fontSize = 18.sp,
+      )
+      LazyColumn(
+        contentPadding = PaddingValues(vertical = 8.dp),
+        modifier = Modifier.fillMaxWidth()
+      ) {
+        itemsIndexed(
+          items = source!!.list.toMutableList(),
```

Comment:
Converting this to a mutable list is not necessary. It's best to have it as an immutable list.